craigslist/base.py

```diff
@@ -410,13 +410,35 @@ def get_list_filters(cls, url):
             cls.__list_filters[url] = utils.get_list_filters(url)
         return cls.__list_filters[url]
 
+    @classmethod
+    def show_categories(cls):
+        url = cls.url_templates["no_area"] % {
+            "site": cls.default_site,
+            "category": cls.default_category,
+        }
+        response = utils.requests_get(url)
+        soup = utils.bs(response.content)
+        cats = [
+            input.get("data-abb")
+            for input in soup.find_all("input", {"class": "catcheck multi_checkbox"})
+        ]
+        cats_title = [a.contents[0] for a in soup.find_all("a", {"class": "category"})]
+
+        print("%s categories:" % cls.__name__)
+        for cat, cat_title in iteritems(dict(zip(cats, cats_title))):
```

Comment:
Wouldn't it be the same to do `zip(cats, cats_title)` without the `dict+iteritems`? Or maybe I'm missing something. Maybe we can add a `sorted` to sort alphabetically.

Reply:
Whoops you are totally right! Brain fart on my end